website/versioned_docs/version-v2.0_alpha/quick-start/poll-finalization.md

```diff
@@ -15,16 +15,13 @@ This could be done using `maci-cli` or by running commands in the `contracts` fo
 pnpm merge:[network] --poll 0
 pnpm run prove:[network] --poll 0 \
     --coordinator-private-key "macisk.1751146b59d32e3c0d7426de411218172428263f93b2fc4d981c036047a4d8c0" \
-    --process-zkey ../cli/zkeys/ProcessMessages_10-2-1-2_test/ProcessMessages_10-2-1-2_test.0.zkey \
-    --tally-zkey ../cli/zkeys/TallyVotes_10-1-2_test/TallyVotes_10-1-2_test.0.zkey \
     --tally-file ../cli/tally.json \
     --output-dir ../cli/proofs/ \
-    --tally-wasm ../cli/zkeys/TallyVotes_10-1-2_test/TallyVotes_10-1-2_test_js/TallyVotes_10-1-2_test.wasm \
-    --process-wasm ../cli/zkeys/ProcessMessages_10-2-1-2_test/ProcessMessages_10-2-1-2_test_js/ProcessMessages_10-2-1-2_test.wasm
+    --start-block 12946802
 ```
 
 :::info
-If the poll was configured to use quadratic voting, please ensure your prove command has the flag `--use-quadratic-voting`. Omitting the flag will make it work with non quadratic voting only.
+If the poll was configured to use quadratic voting, please ensure your prove command has the flag `--use-quadratic-voting`. Omitting the flag will make it work with non quadratic voting only. We also suggest including the `--start-block` flag so the proving doesn't start from block 0.
```

Comment:
you could dig a bit deeper and explain that this part includes fetching all events from the smart contracts and starting from block zero would make it much longer and error prone due to rpc providers limitations